proposals/repo-to-tar/README.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Package Repository to Tar"
+authors: [ "Ashish Kumar <ashishndiitr@gmail.com>" ]
+status: "draft"
+approvers: [ ]
+---
+
+# <Package Repository to Tar>
+
+## Problem Statement
+
+Presently imgpkg does not support the creation of tar file directly.
+In order to obtain a tar file, one needs to first push the bundle image to the registry and then leverage the [command](https://carvel.dev/imgpkg/docs/v0.37.x/air-gapped-workflow/#option-2-with-intermediate-tarball) for copying as tar in the air gapped workflow.
+
+For example, In order to obtain a tar of bundle image of examples/basic-step-2, we need run the following commands in succession :<br>
+
+`imgpkg push -b index.docker.io/user1/simple-app-bundle:v1.0.0 -f examples/basic-step-2` <br>
+`imgpkg copy -b index.docker.io/user1/simple-app-bundle:v1.0.0 --to-tar /tmp/my-image.tar`
+
+## Terminology / Concepts
+Definations of terms used in the proposal with respect to imgpkg can be found [here](https://carvel.dev/imgpkg/docs/v0.37.x/)
+
+## Proposal
+
+#### imgpkg
+imgpkg save command will be added to imgpkg. This command will create a tar file from the bundle image. The command will be used as follows : <br>
+`imgpkg save -b examples/basic-step-2 --to-tar /tmp/my-image.tar`<br>
+It should generate and save tar file at /tmp/my-image.
+
+Now in order to push the tar file to the registry, we can use the already existing copy command as follows : <br>
+`imgpkg copy --tar /tmp/my-image.tar --to-repo registry.corp.com/apps/simple-app-bundle`
+
+#### kctrl
```

Comment:
I think we can make this proposal specific to imgpkg and exclude `kctrl` for the time being.